packages/ui/src/spinner.tsx

```diff
@@ -1,3 +1,4 @@
+/** @jsxRuntime classic */
```

Comment:
would this work instead? https://duncanleung.com/emotion-css-prop-jsx-pragma-storybook/

or maybe this? https://github.com/storybookjs/storybook/issues/12881#issuecomment-715401423 setting it to "classic"?

Reply:
or perhaps this https://github.com/storybookjs/storybook/issues/12952#issuecomment-733987636

Reply:
I tried a few things, but can't remem if any of these specifically ...let me try.

Reply:
I tried these but can't get them to work.

Reply:
This ended up working: https://github.com/storybookjs/storybook/issues/7540#issuecomment-824875458